src/components/timeline/timeline.driver.ts

```diff
@@ -0,0 +1,131 @@
+import {StyleSheet} from 'react-native';
+import {useComponentDriver, ComponentProps} from '../../testkit/new/Component.driver';
+import {TextDriver} from '../text/Text.driver.new';
+import {ImageDriver} from '../image/Image.driver.new';
+import {ViewDriver} from '../view/View.driver.new';
+
+export const TimelineDriver = (props: ComponentProps) => {
+  const driver = useComponentDriver(props);
+
+  const pointDriver = ViewDriver({
+    renderTree: props.renderTree,
+    testID: `${props.testID}.point.container`
+  });
+
+  const labelDriver = TextDriver({
+    renderTree: props.renderTree,
+    testID: `${props.testID}.point.label`
+  });
+
+  const iconDriver = ImageDriver({
+    renderTree: props.renderTree,
+    testID: `${props.testID}.point.icon`
+  });
+
+  const solidTopLineDriver = ViewDriver({
```

Comment:
@Inbal-Tish Since solid and dashed line has different testID's I created these drivers, I don't want it to become messy but we get more clarity when they are separated, WDYT should we merge them under the same View driver ?